cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/TimeSeriesController.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+
+package mil.army.usace.hec.cwms.radar.client.controllers;
+
+import java.io.IOException;
+import java.net.ConnectException;
+import java.time.Instant;
+import mil.army.usace.hec.cwms.http.client.OkHttpUtil;
+import mil.army.usace.hec.cwms.radar.client.ClientNotFoundException;
+import mil.army.usace.hec.cwms.radar.client.HttpUrlProvider;
+import mil.army.usace.hec.cwms.radar.client.NoDataFoundException;
+import mil.army.usace.hec.cwms.radar.client.model.RadarObjectMapper;
+import mil.army.usace.hec.cwms.radar.client.model.TimeSeries;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+public final class TimeSeriesController {
+
+    public TimeSeries retrieveTimeSeries(HttpUrlProvider radarUrlProvider, String timeSeriesId,
+                                         String officeId, String units, String datum,
```

Comment:
datum is vert datum?

Reply:
what is done for non-elev time series?

Reply:
renamed the variable to verticalDatum. If the time series is not elev, the variable is ignored. Currently I'm not aware of how this variable would be used by our existing API's. I believe we always do the conversion client-side to the viewed datum?

Reply:
Oh good point, if this is an elev time series, the serialized vertical datum info should be handed to us by RADAR. @MikeNeilson -- does RADAR's retrieval of Elev time series support vertical datum in the same manner as the database does as required for CWMS modeling, data exchange, etc?

Reply:
I actually noticed that this morning and made the ticket: https://github.com/USACE/cwms-radar-api/issues/101

Reply:
Not sure if it should be part of the timeseries API, or a separate endpoint